android/src/main/java/com/reactcommunity/rndatetimepicker/RNDatePickerDialogFragment.java

```diff
@@ -125,6 +126,14 @@ static DatePickerDialog createDialog(
 
     final DatePicker datePicker = dialog.getDatePicker();
 
+    Integer timeZoneOffsetInMilliseconds = null;
+    if (args != null && args.containsKey(RNConstants.ARG_TZOFFSET_MINS)) {
+      c.setTimeZone(TimeZone.getTimeZone("GMT"));
+      long timeZoneOffsetInMinutesFallback = args.getLong(RNConstants.ARG_TZOFFSET_MINS);
```

Comment:
can this logic be xtracted into a separate fn and the `Integer timeZoneOffsetInMilliseconds = ` would be a one-liner?

thank would be nice, thank you